settings.gradle.kts

```diff
@@ -0,0 +1,34 @@
+include("atomicfu")
+include("atomicfu-transformer")
+include("atomicfu-gradle-plugin")
+include("atomicfu-maven-plugin")
+
+include("atomicfu-native") // this is a kludge so that existing YouTrack config works, todo: remove
+include("integration-testing")
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs") {
+
+            val overridingKotlinVersion =  providers.gradleProperty("community.project.kotlin.version").orNull
```

Comment:
after moving this property here, I have a doubled dependency declaration for `kotlin-script-runtime` in `integration-testing` artifact. It looks like:
```
"dependencies": [
        {
          "group": "org.jetbrains.kotlin",
          "module": "kotlin-stdlib-jdk8",
          "version": {
            "requires": "1.9.21"
          }
        },
        {
          "group": "org.jetbrains.kotlin",
          "module": "kotlin-script-runtime'",
          "version": {
            "requires": "1.9.21"
          }
        },
        {
          "group": "org.jetbrains.kotlin",
          "module": "kotlin-script-runtime",
          "version": {
            "requires": "1.9.21"
          }
        }
      ],
```
I'm not sure in reason, but don't see a big problem here

Reply:
We found a mistake in toml file

Reply:
Oh, with fix we now have identical duplicated blocks